app.json

```diff
@@ -316,6 +316,12 @@
     "OPEN_DISCUSSIONS_FROM_EMAIL": {
       "description": "E-mail to use for the from field"
     },
+    "OPEN_DISCUSSIONS_FRONTPAGE_DIGEST_MAX_EPISODES": {
+      "description": "Max podcast episodes in open digest emails"
```

Comment:
Can you add `"required": false` to each of these since they have default values in Django?